docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -329,11 +313,4 @@ If there is no existing issue for the planned improvements then a new one should
 Look at `existing issues relating to the release process <https://clusterhq.atlassian.net/issues/?jql=labels%20%3D%20release_process%20AND%20status%20!%3D%20done>`_.
 The issue(s) for the planned improvements should be put into the next sprint.
 
-
-.. _wheel: https://pypi.python.org/pypi/wheel
-.. _Google cloud storage: https://console.developers.google.com/project/apps~hybridcluster-docker/storage/archive.clusterhq.com/
-.. _BuildBot web status: http://build.clusterhq.com/boxes-flocker
-.. _virtualenv: https://pypi.python.org/pypi/virtualenv
-.. _Homebrew: http://brew.sh
-.. _CloudFront: https://console.aws.amazon.com/cloudfront/home
```

Comment:
I wanted to move the CloudFront link now it is only used once and then found that in fact many of the other links here are either no longer used or are only used once.